service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocmosisCoverDocumentsService.java

```diff
@@ -33,6 +35,8 @@ DocmosisCoverDocument buildCoverDocumentsData(String familyManCaseNumber,
                             .ccdCaseNumber(formatCCDCaseNumber(caseNumber))
                             .representativeName(representative.getFullName())
                             .representativeAddress(representative.getAddress().getAddressAsString("\n"))
+                            .logoLarge(LOGO_LARGE.getValue())
```

Comment:
could rename it to hmctsLogoLarge & same for small - not fussed if you want to leave though :-)

Reply:
yup, I will change.